assignment/scripts/aboutMe.js

```diff
@@ -46,14 +72,36 @@
 // assigns the highest value to `mostPets`. There's several possibilities --
 // be sure to think through all the scenarios. 
 // console.log `mostPets` after the conditional has run.
+let mostPets;
+if (pets > friendsPets) {
+    mostPets = pets;
+} else if (pets < friendsPets) {
+    mostPets = friendsPets;
+} else if (pets === friendsPets) {
+    mostPets = pets;
+}
+console.log(mostPets);
 
 // 17 - Write a *switch* statement that logs:
 //      "First is the worst" if your lucky number is 1
 //      "Second is the best" if your lucky number is 2
 //      "Third is the one with the polka dot dress" if your lucky number is 3
 //      Otherwise, log "Luck is what happens when preparation meets opportunity"
 //      You'll need to research how to use switch statements!
+// luckyNumber already assigned 2 from earlier.
+switch (luckyNumber) {
+  case 1:
+    console.log('First is the worst')
+    break
+  case 2:
+    console.log('Second is the best')
+    break
+  case 3:
+    console.log('Third is the one with the polka dot dress')
+    break
+  default:
+    console.log('Luck is what happens when preparation meets opportunity')
+}
 
 // 18 -- Rewrite question 13 with a `ternary` operator. You'll need to do research!
-
-
+adventurous ? console.log('Adventures are great!') : console.log('How about we stay home?');
```

Comment:
The key difference between a ternary and an if-block is that a ternary is an _expression_. In other words, it outputs a value. Although this works, a better approach would be to use that output, such as by saving it to a variable.

```js
let message = adventurous ? 'Adventures are great!' : 'How about we stay home?';
console.log(message);